website/content/docs/k8s/multiport/reference/grpcroute.mdx

```diff
@@ -701,8 +701,9 @@ spec:
           groupVersion: v2beta1
           kind: Service
         name: api
-      # The configuration targets the workload port, not the service port.
-      port: "api-workload"
+      # The virtual port number for the "api-workload" target port.
+      # If not using virtual ports, specify the workload port name directly.
```

Comment:
This might be verbose to include inline everywhere - I'm open to suggestions about how to factor it out into docs and leave a shorter note here if that's preferable.